### DISCUSSION TITLE
Tunneling extension

### DISCUSSION THREAD

**Discussion by Lewis-Di39** [TheLastBen/fast-stable-diffusion#1662] [Ideas]
Dear developer Ben

I am writing to express my appreciation for your work on Automatic1111 on sd-webui. I have found it to be an incredibly useful tool for my work. I would like to suggest adding a tunneling extension to Automatic1111, as it seems to work faster than Gradio. I believe this would greatly enhance the usability and functionality of the tool, and I would love to see this feature added in the future. Thank you for your time and consideration.

Best regards

**Comment by ghost** [comment]
If the tunneling you were referring to is [localtunnel](https://github.com/localtunnel/localtunnel), I recently created [sd-webui-localtunnel-client](https://github.com/jnyfil/sd-webui-localtunnel-client).
Use it together with [localtunnel-loopback](https://github.com/jnyfil/localtunnel-loopback)

**Reply by gilroff** [reply]
whats the benefit of this?
